app/lib/system/deploy.rb

```diff
@@ -71,8 +70,8 @@ def self.call(_)
 
     def self.load_info!(deploy_info = parse_deploy_info)
       self.info = Info.new(deploy_info)
-    rescue StandardError => error
-      self.info = InvalidInfo.new(error)
+    rescue StandardError => e
```

Comment:
why on earth would rubocop say that `e` is clearer than `error` ? 😕

Reply:
It's not about clarity but naming convention. We could change it to `error` or disable it altogether.